.infra/Pulumi.prod.yaml

```diff
@@ -110,8 +110,7 @@ config:
     onesignalAppId:
       secure: AAABALTizkxZkCfsq6/fx2v7nXxsVP7psWBfSUct0zeyrX0ABxPpD30zIsyx5r5wqt1UJIIXDIVOnhM7W3vzrzaatrE=
     otelEnabled: true
-    otelTracesSampler: parentbased_traceidratio
-    otelTracesSamplerArg: 0.05
```

Comment:
Where did sampling go?

Reply:
`parentbased_always_on`
So API always samples, and sends it to the collector, which then decides if out should be sampled or not (tail sampling)
That way we can enable 100% sampling for staff or errors, all others have a percentage rate

Reply:
See here https://github.com/dailydotdev/daily-infra/blob/7dad00fd2c19cc45f7999b48591f58b318b020a0/tracing.ts#L71-L87

Reply:
my next question was what `parentbased_traceidratio` does 😆 , cool!

Reply:
`parentbased` it takes into account what the sampling rate is set on the parent. I.e. if API was called from a different service that was also doing tracing, but it decided to not sample this request. That would make it respect that.

`traceidratio` just sampling rate.

`always_on` ... always on

`parentbased_traceidratio` Respects parent decision, but if none, sets sampling rate

`parentbased_always_on` Respects parent decision, but if none, always on